src/rulesets/jsonapi.ts

```diff
@@ -77,7 +77,7 @@ export const rules = {
     );
   },
   contentType: ({ responses }: SnykApiCheckDsl) => {
-    responses.requirement.must(
+    responses.requirementOnChange.must(
```

Comment:
this is probably fine. It might ask people to add support for your custom JSON content type if they don't already have it. 

If we only want to enforce this on new surface area use added